tests/unit_tests/test_constraints.py

```diff
@@ -153,17 +153,17 @@ def test_constraints_NUM_FLOAT():
 def test_constraints_DATE_AFTER():
     constraint = Constraint(constraint_dict=mock_constraints.CONSTRAINT_DATE_AFTER)
 
-    assert constraint.apply({'currentTime': datetime(2022, 1, 23)})
-    assert not constraint.apply({'currentTime': datetime(2022, 1, 22)})
-    assert not constraint.apply({'currentTime': datetime(2022, 1, 21)})
+    assert constraint.apply({'currentTime': datetime(2022, 1, 23, tzinfo=pytz.UTC)})
```

Comment:
Patching tests to deal with the behaviour change - this SDK was incorrectly ignoring timezones